packages/components/tests/dummy/app/templates/components/app-footer.hbs

```diff
@@ -0,0 +1,207 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "AppFooter Component"}}
+
+<Shw::Text::H1>AppFooter</Shw::Text::H1>
```

Comment:
I would suggest to add to the showcase a section in which you test the responsiveness and stacking of the element at different width of the parent, to
- make sure the implementation in code is solid and reflects the expected behaviour
- visualize (for us, and for other stakeholders) how it works in code, how it's rendered in a browser
- make sure that is covered by visual regression testing

see the "Content wrapping" section here as an example
- https://hds-showcase.vercel.app/components/page-header
- https://github.com/hashicorp/design-system/blob/main/packages/components/tests/dummy/app/templates/components/page-header.hbs#L296-L332

(you can also take directly the code ;) )

Reply:
I pushed up "Layout" examples that I've been using to test with in the Showcase.